web/remote/remote.go

```diff
@@ -30,7 +33,8 @@ func remoteGet(c echo.Context) error {
 
 func remotePost(c echo.Context) error {
 	doctype := c.Param("doctype")
-	if err := middlewares.AllowWholeType(c, permission.POST, doctype); err != nil {
+	slug, err := allowWholeType(c, permission.POST, doctype)
```

Comment:
nit: I don't see how this could be done otherwise but it feels a bit weird to get the slug of the requesting app from the permission rather than the context itsellf